web.go

```diff
@@ -160,6 +167,9 @@ type Handler struct {
 	// This feature may be used to further customise requests or
 	// as a filter or as an extended http.HandlerFunc.
 	Func func(http.ResponseWriter, *http.Request, *Claims) (pass bool)
+
+	// When true this will processResult the output
+	Jsonify bool
```

Comment:
// Write auth error bodies as JSON (instead of plain text).
// The message property has a description line.
ErrAsJSON bool

Reply:
Yeah, that's a good idea. I'll change this.

> Out of curiosity: why do you want JSON bodies? Does that help with some JavaScript framework?

These changes were done as a quick fix for my REST implementation. There are plenty of JWT libraries, but I found that your library is good maintained and quite simple to understand. One of the requirements was, that all endpoints must return valid JSON value, thus I have to modify sources, and thought, that I might not be the only one with these requirements for services.